feat: change docs for token revocation (#1964)
buildkite/docs: pages/apis/managing_api_tokens.md

Click through the token you'd like to remove, then click the 'Remove Organization from Token' button.

> Removing access from a token will send a notification email to the token's owner, and they will not be able to re-add your organization to the token's scope.

Reviewer: Consider tightening:

```suggestion
Removing access from a token sends a notification email to the token's owner, who cannot re-add your organization to the token's scope.
```

Author: 👍

### Can I re-add my organization to a token?

> If an organization has revoked a token, it cannot be re-added to the token. The token owner would have to create a new token with access to your organization.

Reviewer: Should we provide a simple "no" upfront?

```suggestion
No. If an organization has revoked a token, it cannot be re-added to the token. The token owner would have to create a new token with access to your organization.
```

Author: changed
